pilot/CreateViewController.swift

```diff
@@ -48,7 +48,7 @@ class CreateViewController: UIViewController {
             activitySpinner.startAnimating()
             signUpButton.isEnabled = false
             
-            PilotUser.upload(with: ThunderRouter.createPilotUser(pilotUser), onSuccess: {
+            PilotUser.upload(with: ThunderRouter.createPilotUser(pilotUser), onSuccess: { pilotUser in
```

Comment:
Since you don't use the variable in the closure you can use `_ in`

Reply:
Good point